live_pipeline/extract/test_extract.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from extract import get_plants, APIError
+import requests_mock
```

Comment:
Local imports should appear after Python imports as per pep8 requirements https://peps.python.org/pep-0008/#imports